Recipes/Recipes.DataAccess/Repositories/RecipesRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Recipes.DataAccess.Entities;
+
+namespace Recipes.DataAccess.Repositories
+{
+    public class RecipesRepository : GenericRepository<RecipeEntity>, IGenericRepository<RecipeEntity>
+    {
+        public RecipesRepository(RecipesDatabaseContext context) : base(context) { }
+
+        public override async Task<IEnumerable<RecipeEntity>> GetAllAsync()
+        {
+            return await Context.Set<RecipeEntity>().Include(s => s.Tags).ThenInclude(s => s.Tag).ToListAsync();
+        }
+
+        public override async Task<RecipeEntity?> GetByIdAsync(params object[] id)
```

Comment:
You claim this is an async method, but it doesn't feature any async code? Curious..

Reply:
this overrides a method that was async. This is fake async :)

Reply:
changed original method to not be async